aten/src/TH/THAllocator.cpp

```diff
@@ -496,16 +496,23 @@ int THRefcountedMapAllocator::decref()
 
 THRefcountedMapAllocatorArgCheck::THRefcountedMapAllocatorArgCheck(int flags) {}
 
-THRefcountedMapAllocator::THRefcountedMapAllocator(const char *filename, int flags, size_t size) {
+THRefcountedMapAllocator::THRefcountedMapAllocator(const char *filename, int flags, size_t size)
+  : THRefcountedMapAllocatorArgCheck(flags),
+    THMapAllocator(filename, flags, size + TH_ALLOC_ALIGNMENT)
+{
   AT_ERROR("refcounted file mapping not supported on your system");
 }
 
-THRefcountedMapAllocator::THRefcountedMapAllocator(WithFd, const char *filename, int fd, int flags, size_t size) {
+THRefcountedMapAllocator::THRefcountedMapAllocator(WithFd, const char *filename, int fd, int flags, size_t size)
+  : THRefcountedMapAllocatorArgCheck(flags),
+    THMapAllocator(WITH_FD, filename, flags, fd, size + TH_ALLOC_ALIGNMENT)
```

Comment:
Those calls seem unnecessary given that we raise an error anyway?

Reply:
You are right that these calls should not be called - but seems we still need make these changes to make it compile. In some earlier diff we changed the base class which no longer takes default constructor - other call sites were updated accordingly but this one was missed and it's not covered by integration test.

Reply:
Oh, I see. Makes sense!